ceno_zkvm/src/e2e.rs

```diff
@@ -45,6 +45,7 @@ pub struct EmulationResult {
     pi: PublicValues<u32>,
 }
 
+// TODO(Matthias): handle hints properly.
```

Comment:
This is something to fix soon.  But I don't know enough at the moment.

It's definitely broken in `master`, though.